hack/create_self_signed_cert.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ex
+
+TMPDIR={$TMPDIR:-"/tmp/k8s-webhook-server/serving-certs"}
+SERVICE={$SERVICE:-"glance"}
+NAMESPACE={$NAMESPACE:-"openstack"}
```

Comment:
NAMESPACE=${NAMESPACE:-"openstack"}

Reply:
sorry that was typo... thanks for noticing. Fixed.

Reply:
+1 Thanks!